src/ApiLoader/EnableInTraceeWindows.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// clang-format off
```

Comment:
Why has `EnableInTracee.h` to be included first? I looked into the header but found no apparent reason. Maybe add a comment. Same above.

Reply:
It was really just to respect the convention of including the associated header first.

Reply:
Oh, right and the `// clang-format off` is just needed since there are specific implementations, got it.